osmaxx/core/templates/pages/about_us.html

```diff
@@ -18,7 +18,7 @@
     <h3>Useful links</h3>
     <p>Can be found in our separate <a href="{% url 'pages:downloads' %}">downloads page</a></p>
 
-    <h3>About Us</h3>
+    <h3>About us</h3>
```

Comment:
This isn't a [sentence](http://grammar.about.com/od/grammarfaq/f/capitalstitle.htm), so capitalization is IMHO correct.